cloud/core/README.md

```diff
@@ -0,0 +1,80 @@
+# Cloud Core API
+
+This is a quick and dirty OpenAPI-oriented JSON-over-HTTP service for
+configuring our cloud core stuff, written in Python.
+
+# Nix
+
+## Build
+
+We need to build a Python interpreter which can import all of the various
+modules that we want to use.
+
+If you have Nix installed with Nix flakes enabled, then:
+
+```bash
+$ nix build .#bravo -o bravo && export DOCS_PATH=bravo
+$ nix build
+```
+
+This will both build the Python environment and also set up a `bravo` folder
+for docs.
+
+If Nix flakes aren't enabled, then it's still possible but requires flags:
+
+```bash
+$ nix --extra-experimental-features nix-command --extra-experimental-features flakes build .#bravo -o bravo
+$ nix --extra-experimental-features nix-command --extra-experimental-features flakes build
+```
+
+Now the resulting interpreter can be used as if from a virtualenv:
+
+```bash
+$ result/bin/uvicorn --host=0.0.0.0 --port=8000 core:app
+```
+
+# Podman/Docker
+
+If Nix isn't installed, there is a Podman-compatible Dockerfile which can be
+used to build and run the entire service at once. The Dockerfile executes the
+same `nix build` and `uvicorn` commands as above, but within a container.
+
+## Fetch example docs
+
+There's several possible documentation sources that you can try. Here's one
+from my old Minecraft server, Bravo:
+
+```bash
+$ git clone https://github.com/bravoserver/bravo
```

Comment:
I want our documentation and code to be fairly consistent. I understand the value of `$` in instructions, and I understand the value of no `$`. I really don't have a strong opinion - although `$` may be better. For consistency across our documentation do you have a strong opinion?